src/base/static/components/molecules/map-legend-item.js

```diff
@@ -1,23 +1,56 @@
 import React from "react";
 import PropTypes from "prop-types";
-import classNames from "classnames";
+import styled from "react-emotion";
 
-import { Icon, Swatch } from "../atoms/feedback";
-import { PanelLabel } from "../atoms/typography";
+import { Icon } from "../atoms/feedback";
+import { RegularLabel } from "../atoms/typography";
 
 import "./map-legend-item.scss";
 
-const MapLegendItem = props => (
-  <div className="map-legend-item">
-    {props.icon && (
-      <div className="map-legend-item__icon-container">
-        <Icon icon={props.icon} classes="map-legend-item__icon" />
-      </div>
-    )}
-    {props.swatch && <Swatch color={props.swatch} />}
-    <PanelLabel classes="map-legend-item__label">{props.label}</PanelLabel>
-  </div>
-);
+const ColorSwatch = styled("span")(props => ({
+  backgroundColor: props.color,
+  width: 20,
+  height: 20,
+  marginRight: 10,
+}));
+
+// TODO: Abstract these components out when we refactor other panel types.
+const IconContainer = styled("div")(props => ({
+  display: "flex",
+  justifyContent: "center",
+  flex: 1,
+}));
+
+const MapLegendItemContainer = styled("div")(props => ({
+  display: "flex",
+  alignItems: "center",
+  marginBottom: 5,
+  paddingLeft: 15,
+}));
+
+const MapLegendItem = props => {
```

Comment:
It wasn't clear to me how to make the export a styled component. So I made the `MapLegendItemContainer`, but it feels like I'm missing a better way to structure this.

Reply:
Why does the export need to be a styled component?

I think the use of `MapLegendItemContainer` makes sense here.